mssql_insert_block.py

```diff
@@ -1,12 +1,13 @@
-from nio.properties import VersionProperty, Property
+from nio.properties import VersionProperty, Property, StringProperty
 from .mssql_base import MSSQLBase
 from nio.block.mixins.enrich.enrich_signals import EnrichSignals
 
 
 class MSSQLInsert(EnrichSignals, MSSQLBase):
 
-    version = VersionProperty("0.2.0")
-    row = Property(title='Row', default='{{ $.to_dict() }}')
+    version = VersionProperty("1.0.0")
+    table = StringProperty(title='Table', default='{{ $table }}', order=5)
```

Comment:
i feel like these numberings are all off since a lot of stuff has been extracted into the `Connection` property holder.